lib/donations/currencies.rb

```diff
@@ -0,0 +1,47 @@
+# == Currency Converter to JSON
+#
+# Takes an array of integer amounts, in US cents, and
+# converts to GBP and EUR.
+#
+# ==== Examples
+#
+#   currencies = Donations::Currencies.for(102, 204, 302, 401)
+#   currencies.to_hash # also to_json
+#   =>
+#   {
+#     USD: [1.02, 2.04, 3.02, 4.01],
```

Comment:
Ah overlapped with Eric. Yep, I wrote in an email re donation design that the edge case of 1 USD and 2 USD (for example) both rounding to 1 GBP was one we need to handle. In that case, we should display 1 GBP and 2 GBP, although 3 USD might round to 2 GBP, so then we need to make sure that would display as 3 GBP, etc. That is, don't really have firm views on how handles, but needs logic never to display two identical donation amounts.